samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/CalculatedQuestionVariableBean.java

```diff
@@ -144,4 +145,12 @@ public String getMatch() {
         return match;
         
     }
+
+    public boolean isFromui() {
```

Comment:
Do you think we can use @setter and @getter from lombok using the final variable name?

Reply:
Changed to lombok